packages/libs/eda/src/lib/core/components/filter/HistogramFilter.tsx

```diff
@@ -596,11 +578,7 @@ function HistogramPlotWithControls({
         {}, // no overrides
         true // use inclusive less than or equal to for min
       ),
-    [
-      defaultUIState.independentAxisRange,
-      defaultDependentAxisRange,
-      uiState.dependentAxisRange,
-    ]
+    [defaultUIState.independentAxisRange, dependentAxisMinPosMaxRange, uiState]
```

Comment:
These dependencies were all wrong - yellow truncation stuff seems to work fine still after the fix.